tello Test/tello_web_server.py

```
@@ -799,6 +1550,283 @@ async def rotate_handler(request: web.Request) -> web.Response:
+        logger.error(f"CopilotKit API error: {e}")
+        return web.json_response({
+            "success": False,
+            "error": str(e),
+            "timestamp": datetime.now().isoformat()
+        }, status=500)
+
+async def call_mastra_agent(message: str, thread_id: str, resource_id: str) -> str:
+    """Mastraエージェントを呼び出す"""
+    try:
+        import aiohttp
+        import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)